src/mds/MDCache.cc

```diff
@@ -206,6 +206,15 @@ void MDCache::handle_conf_change(const ConfigProxy& conf,
     cache_inode_limit = g_conf().get_val<int64_t>("mds_cache_size");
   if (changed.count("mds_cache_memory_limit"))
     cache_memory_limit = g_conf().get_val<Option::size_t>("mds_cache_memory_limit");
+  if (changed.count("mds_memory_target"))
+    cache_memory_limit = g_conf().get_val<Option::size_t>("mds_memory_target");
```

Comment:
The `mds_memory_target` should be a class member of `MDSRank`. It applies to all MDS memory, not just the cache.

Reply:
Agreed. Will move it to `MDSRank`